apps/site/i18n.tsx

```diff
@@ -1,3 +1,4 @@
+import { importLocale } from '@node-core/website-i18n';
```

Comment:
Here's the specific error in play:

```plaintext
@node-core/website:lint:js: /Users/josh/repos/nodejs.org/apps/site/i18n.tsx
@node-core/website:lint:js:   1:30  error  Unable to resolve path to module '@node-core/website-i18n'  import/no-unresolved
```

We could go down a whole rabbit hole of configuring the `import` plugin's `resolvers`: https://github.com/import-js/eslint-plugin-import?tab=readme-ov-file#resolvers 

But, this rule isn't very useful when you're using TypeScript. TypeScript is already checking your `import` paths! So, my advice would be to just `"import/no-unresolved": "off"` in the ESLint config. https://typescript-eslint.io/troubleshooting/typed-linting/performance#eslint-plugin-import